services/pull/review.go

```diff
@@ -43,6 +43,23 @@ func (err ErrDismissRequestOnClosedPR) Unwrap() error {
 	return util.ErrPermissionDenied
 }
 
+// ErrSubmitReviewOnClosedPR represents an error when an user tries to submit an approve or reject review associated to a closed or merged PR.
+type ErrSubmitReviewOnClosedPR struct{}
+
+// IsErrSubmitReviewOnClosedPR checks if an error is an ErrSubmitReviewOnClosedPR.
+func IsErrSubmitReviewOnClosedPR(err error) bool {
```

Comment:
Maybe we can use `errors.As` as much as possible? Many `IsErrXxx` were written for old Golang.

Reply:
Gotcha. After looking at the `errors` package I decided to use `errors.Is` and just turn `ErrSubmitReviewOnClosedPR` to a globally-accessible error object as this seems to be a similar pattern in other areas of the codebase and on top of this I don't need to do anything complex when handling this error.

Reply:
Golang has a bad error system ....

`Is` checkes "value", eg: `var MyErr = errors.New("...")`

`As` checks the "type", eg: `var MyErr = MyErrStruct{}`

Not easy to use, so I edited my suggestion, either is fine to me.